bach/tests/functional/bach/test_series_string.py

```diff
@@ -110,3 +110,24 @@ def test_get_dummies(engine) -> None:
             [3, 1, 0, 0]
         ],
     )
+
+
+def test_string_replace(engine) -> None:
+    bt = get_df_with_test_data(engine)
+    municipality = bt['municipality'].sort_index()
+    assert isinstance(municipality, SeriesString)
+
+    municipalities = municipality.to_numpy()
+    replace_pat = ['-', 'west', 'ee', 'ú', 'â']
```

Comment:
```suggestion
    replace_pat = ['-', 'west', 'ee', 'ú', 'â', 'e']
```
Having at least one case where we replace more than one occurrence in the string seems like a good idea to me